tools/runstrat/tests/test_pipeline.py

```diff
@@ -0,0 +1,25 @@
+from pathlib import Path
+import os
+
+from runstrat import Args, entrypoint, RunMode
+from src.psstrategy import BasePSStrategy
+
+ARTIFACTS_DIR = "artifacts"
+os.makedirs(ARTIFACTS_DIR, exist_ok=True)
+
+
+def test_pipeline_with_mock_data():
+    args = Args(
+        strategy=BasePSStrategy(name="ExecutionTreeContributedCoverage"),
```

Comment:
```suggestion
        strategy= ExecutionTreeContributedCoverageStrategy(name="ExecutionTreeContributedCoverage"),
```

I intended BasePSStrategy to be abstract